### PR TITLE
add skipped_read_repairs to legacy stat map

### PR DESCRIPTION
**Pull request by uwiger**

### CODE REVIEW

**Comment by jburwell** [comment]
:+1: 0eee998

**Comment by jburwell** [comment]
@borshop merge

File: src/riak_core_stat.erl [hunk 1]
```diff
@@ -352,6 +352,8 @@ legacy_stat_map() ->
                                             {max   , node_get_fsm_counter_objsize_100}]},
      {[riak_kv,node,gets,read_repairs], [{one, read_repairs},
                                          {count, read_repairs_total}]},
+     {[riak_kv,node,gets,skipped_read_repairs], [{one, skipped_read_repairs},
```
Comment (jrwest) [inline src/riak_core_stat.erl:355]: a curiosity: why are we referencing `riak_kv` from `riak_core`? Is this legacy code or something we introduced via exometer?

Comment (russelldb) [inline src/riak_core_stat.erl:355]: Yeah. Shouldn't this be in riak_kv_stat_bc?
On 21 Nov 2014 16:18, Jordan West notifications@github.com wrote:In src/riak_core_stat.erl:

> @@ -352,6 +352,8 @@ legacy_stat_map() ->
>                                              {max   , node_get_fsm_counter_objsize_100}]},
>       {[riak_kv,node,gets,read_repairs], [{one, read_repairs},
>                                           {count, read_repairs_total}]},
> -     {[riak_kv,node,gets,skipped_read_repairs], [{one, skipped_read_repairs},

a curiosity: why are we referencing riak_kv from riak_core? Is this legacy code or something we introduced via exometer?

Comment (jburwell) [inline src/riak_core_stat.erl:355]: @seancribbs, @ulf, and myself made the decision to temporarialy duplicate the legacy_stat_map was copied into riak_core temporarily to complete the Exometer merge.  @ulf will be removing this as part of post-merge cleanup in the next week or so now that Exometer provides a registration system to track these mappings internally.

**Pull request merged**